pixart.js

```diff
@@ -1 +1,37 @@
+$(document).ready(function(){
+  $(".brush").css("background-color","red");
```

Comment:
This works, but why is this isolated from the object? Could add as another property on the `pixArt` object